Library/ViewModels/PostCampaignCheckoutViewModel.swift

```diff
@@ -514,3 +539,26 @@ public class PostCampaignCheckoutViewModel: PostCampaignCheckoutViewModelType,
   public var inputs: PostCampaignCheckoutViewModelInputs { return self }
   public var outputs: PostCampaignCheckoutViewModelOutputs { return self }
 }
+
+private func paymentMethodValid(
+  project: Project,
+  reward: Reward,
+  paymentSource: PaymentSourceSelected,
+  context: PledgeViewContext
+) -> Bool {
+  guard
+    let backedPaymentSourceId = project.personalization.backing?.paymentSource?.id,
+    context.isUpdating,
+    userIsBacking(reward: reward, inProject: project)
+  else {
+    return true
```

Comment:
Won't this always return true if you haven't backed this project already?

Reply:
oh. yeah I guess that it true. I keep forgetting we won't be updating late pledges